backend/tracim_backend/app_models/validator.py

```diff
@@ -14,8 +14,8 @@
 from tracim_backend.app_models.contents import content_type_list
 from tracim_backend.applications.agenda.models import AgendaType
 from tracim_backend.exceptions import TracimValidationFailed
+from tracim_backend.lib.utils.dict_parsing import validate_simple_dict
```

Comment:
This is unused, isn't it?

Reply:
Linters do not complain about this?

Reply:
This is used for validation of user_config.